docs/api/browser-window.md

```diff
@@ -169,6 +169,15 @@ Emitted when window is minimized.
 
 Emitted when window is restored from minimized state.
 
+### Event: 'resize'
+
+Emitted when window is getting resized on all platforms.
+
+### Event: 'move'
```

Comment:
I think it should be after it has `moved`/`resized` on all platforms. If people really need it live, which is IMHO a bad idea as it will be very expensive and chatty, it should be separate events.